devicon.json

```diff
@@ -3145,6 +3145,32 @@
         "color": "#f24e1e",
         "aliases": []
     },
+    {
+        "name": "filamentphp",
+        "tags": [
+            "framework",
+            "laravel"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#000000",
+        "aliases": [
```

Comment:
Close but an array is not supported. Each alias will need to be its own object. 

```
        "aliases": [
            {
                "base": "original",
                "alias": "original-wordmark"
            }, 
             {
                "base": "original",
                "alias": "plain"
            },
             {
                "base": "original",
                "alias": "plain-wordmark"
            } 
        ]
```

Reply:
Okay, I thought it would save space & size :)